dask/dataframe/tests/test_arithmetics_reduction.py

```diff
@@ -1333,7 +1341,11 @@ def test_reductions_frame_dtypes_numeric_only_supported(func):
         getattr(df, func)(numeric_only=True),
         getattr(ddf, func)(numeric_only=True),
     )
-    errors = TypeError if pa is None else (TypeError, ArrowNotImplementedError)
+    errors = (
+        (ValueError, TypeError)
+        if pa is None
+        else (ValueError, TypeError, ArrowNotImplementedError)
+    )
```

Comment:
I'm guessing the additional error also depends on `DASK_EXPR_ENABLED`, is that right? Is it worth making this conditional as well? (My guess is no. :) )

Reply:
Probably not no, the tests was already ugly before this pr :)